src/main/java/racing/domain/Car.java

```diff
@@ -21,11 +23,16 @@ public int getMovingRange() {
         return movingRange;
     }
 
-    public void moved() {
-        movingRange += 1;
+    public Car getRandomMove(int random, Car car) {
+        if (random > MOVE_NUMBER) {
+            return new Car(name, movingRange + MOVE_COUNT);
+        }
+        return car;
```

Comment:
객체의 응집도가 점점 높아지는 것 같아요 👍 
```suggestion
    public Car getRandomMove(int random) {
            if (random > MOVE_NUMBER) {
                return new Car(name, movingRange + MOVE_COUNT);
            }
            return this;
    }
```
위와 같이 `this` 를 활용하면 파라미터 하나를 더 줄일 수 있을 것 같네요!!